shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -521,6 +522,7 @@ \subsection{Pool Reaping Transition}
           ~ \\
           \varUpdate{\var{retired}} & \varUpdate{\subtractdom} & \varUpdate{\var{stpools}} \\
           \varUpdate{\var{retired}} & \varUpdate{\subtractdom} & \varUpdate{\var{poolParams}} \\
+          \varUpdate{\var{retired}} & \varUpdate{\subtractdom} & \varUpdate{\var{fPoolParams}} \\
```

Comment:
It's worth noting that removing retirees from fPParams in POOLREAP is benign because fPParams is emptied out in EPOCH just before calling POOLREAP.

Reply:
indeed